cwms-data-api/src/main/java/cwms/cda/data/dto/location/kind/TurbineChange.java

```diff
@@ -45,8 +45,15 @@
 @JsonPropertyOrder({"project-id", "change-date", "protected", "discharge-computation-type", "reason-type", "notes"})
 public final class TurbineChange extends PhysicalStructureChange<TurbineSetting> {
 
+    private Double poolElevation;
+
     TurbineChange(Builder builder) {
         super(builder);
+        poolElevation = builder.poolElevation;
```

Comment:
This is getting set in `PhysicalStructureChange` as well (and you're adding the getter to this class), does it belong in both?

Reply:
Forgot to delete those I think